.github/workflows/ci.yml

```
@@ -53,14 +53,14 @@ jobs:
       
       - name: Test binary execution
         run: |
-          chmod +x dist/private-journal-mcp
-          ./dist/private-journal-mcp --help || true
+          chmod +x dist/agent-journal
+          ./dist/agent-journal --help || true
       
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)